python/shark_turbine/aot/compiled_module.py

```diff
@@ -429,6 +431,39 @@ def run_pass_pipeline(
                 print(module_op, file=sys.stderr)
                 raise
 
+    def save_vmfb(inst: "CompiledModule", path: Union[Path, str]):
+        flags = [
+        "--iree-input-type=torch",
+        "--iree-vm-bytecode-module-output-format=flatbuffer-binary",
+        "--mlir-print-debuginfo",
+        "--mlir-print-op-on-diagnostic=false",
+        "--iree-llvmcpu-target-cpu-features=host",
+        "--iree-llvmcpu-target-triple=x86_64-linux-gnu",
+        "--iree-llvmcpu-enable-microkernels",
+        "--iree-llvmcpu-stack-allocation-limit=256000",
+        "--iree-stream-resource-index-bits=64",
+        "--iree-vm-target-index-bits=64",
+        "--iree-vm-bytecode-module-strip-source-map=true",
+        "--iree-util-zero-fill-elided-attrs",
+        "--iree-vm-target-truncate-unsupported-floats",
+        "--iree-codegen-check-ir-before-llvm-conversion=false",
+        "--iree-vm-bytecode-module-output-format=flatbuffer-binary",
+        "--iree-opt-const-expr-hoisting=False",
```

Comment:
So these flags are something I copy pasted offhandedly from a vicuna.py custom compile command in a google doc somewhere, I don't think they're a canonical solid list to just make a default for all compiled models in turbine. Some of them are probably fine but probably worth looking through.